verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.ts

```diff
@@ -23,7 +23,7 @@ describe('Linelist table', function () {
         cy.contains('www.example.com');
     });
 
-    it('Can edit a case', function () {
+    it.only('Can go to the edit page', function () {
```

Comment:
Oh cool does it.only run only that test? Good to know, but should be removed :)

Reply:
yeah it does :) thanks for spotting! removed...